server/grpc.go

```diff
@@ -101,35 +112,65 @@ func (ss *StreamServer) Run() {
 				metrics.DecConnection()
 			case payload := <-ss.payloads:
 				ss.clientManager.SendPayload(payload)
+			case re := <-ss.resfreshEvents:
+				ss.clientManager.DeleteEvents(re.client)
+				re.client.SetEvents(re.events)
+				ss.clientManager.AddClient(*re.client)
 			}
 		}
 	}()
 }
 
-func (ss *StreamServer) Events(request *proto.Request, es proto.StreamService_EventsServer) error {
-	ss.accessLogger.Info("gRPC",
-		zap.Array("request-events", zapcore.ArrayMarshalerFunc(func(enc zapcore.ArrayEncoder) error {
-			for _, e := range request.Events {
-				enc.AppendString(e.Type)
+func (ss *StreamServer) Events(es proto.StreamService_EventsServer) error {
```

Comment:
__[golint]__
* [golint] exported method StreamServer.Events should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->